### PR TITLE
Power supply

### PR DESCRIPTION
**Pull request by bkstephens1**
Updated Power Supply detail design. Not to be actually bought as we moved away from DC injection braking and we are now using a VFD. Was done for completeness.

### CODE REVIEW

**Comment by bkstephens1** [comment]
I accidently did this wrong.

**Comment by bkstephens1** [comment]
Sorry for closing the request. I thought it did not have the proper .md file

File: Documentation/Signoffs/Power Supply.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+## Functionality ##
+This subsystem is to supply the current to the motor’s used in the braking subsystem. The user will be able to change the amount of current that is entered into the motor winding away the system. It will have a power supply going into a buck converter and the current will change depending on the duty cycle of the MOSFET which is used as a switch. The MOSFET will be switched on and off from PWM of an Arduino Nano with a frequency of 62.5 kHz and a PWM resolution of 8 bits which gives us 256 levels. This allows us to adjust the duty cycle in 0.391% increments. The user will set the duty cycle, this will cause the current to increase as the duty cycle decreases and to decrease as the duty cycle increases.
```
Comment (micahr234) [inline Documentation/Signoffs/Power Supply.md]: Just give the overview... save the detail until the analysis.

File: Documentation/Signoffs/Power Supply.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+![image](https://github.com/user-attachments/assets/0b3d8e39-7646-40e5-ba88-f462739e8e50)
+
+Above: Basic Topology of Buck Converte
+
+
+## Constraints ##
+| No. | Constraints                                                                                     | Origin           |
+|-----|-------------------------------------------------------------------------------------------------|------------------|
+| 1   | Must not exceed 18.4375 amps       | Design Constraint |
```
Comment (micahr234) [inline Documentation/Signoffs/Power Supply.md]: What must not exceed 18.4 Amps??? Be specific.

File: Documentation/Signoffs/Power Supply.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+To determine the minimum input capacitor size the following equation is used. [2]
+
+![image](https://github.com/user-attachments/assets/ac1dcf1c-0487-4b38-8588-cf4adbfbc5b9)
+
+Having the input votlage ripple to be 100mV we have minimum input capacitor size of 135.8 microfarad.
+
+To assist with power dissipation in the diodes two Schtokky diodes in paralle were used. The forward voltage is 0.84 which causes the power dissipation to be (16.871/2)*(0.84) = 7.09W.
+
+The arduino nano was chosen as it allows for 5V, 8 bit PWM on each PWM pin. Since the clock is 16Mhz, an 8 bit PWM resolution allows for a frequency of 62.5kHz. The 8 bit PWM allows for a duty cycle resolution 0.4% which will allow for greater control.
```
Comment (micahr234) [inline Documentation/Signoffs/Power Supply.md:67]: How are you going to control it from the user interface???

File: Documentation/Signoffs/Power Supply.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+Allowing for the voltage ripple to be 100mV the minimum capacitor size is 88 microfarad. As the output capacitor size increases the amount of voltage ripple will decrease.
+
+To determine the minimum input capacitor size the following equation is used. [2]
+
+![image](https://github.com/user-attachments/assets/ac1dcf1c-0487-4b38-8588-cf4adbfbc5b9)
+
+Having the input votlage ripple to be 100mV we have minimum input capacitor size of 135.8 microfarad.
+
+To assist with power dissipation in the diodes two Schtokky diodes in paralle were used. The forward voltage is 0.84 which causes the power dissipation to be (16.871/2)*(0.84) = 7.09W.
```
Comment (micahr234) [inline Documentation/Signoffs/Power Supply.md]: Are they able to dissipate this much power?

**Review by micahr234 — COMMENTED** [review]
Add PCB layout

**Review by micahr234 — APPROVED** [review]

**Pull request merged**